lib/gossip/dissemination.js

```diff
@@ -182,7 +194,7 @@ Dissemination.prototype._issueAs = function _issueAs(filterChange, mapChanges) {
             continue;
         }
 
-        if (change.piggybackCount > this.maxPiggybackCount) {
+        if (change.piggybackCount >= this.maxPiggybackCount) {
```

Comment:
I'm not sure I understand. The piggyback counter is only bumped when the communication is successful (with the exception on issueAsReceiver).
Previously the counter was  raised before this statement, in that case it was possible to throw the change away when we didn't have confirmation that the change was successfully disseminated.